.eslintrc

```diff
@@ -1,177 +1,177 @@
 {
```

Comment:
we should extend `eslint-config-wolox` if necessary we should create an angular config